New warnings with the latest libtommath

With the latest develop branch of libtommath I see the following warnings:
```
cc -Isrc/headers -Itests -DLTC_SOURCE -Wall -Wextra -O3 -DUSE_LTM -DLTM_DESC -I../libtommath -c src/math/ltm_desc.c -o src/math/ltm_desc.o
src/math/ltm_desc.c: In function ‘set_int’:
src/math/ltm_desc.c:92:4: warning: ‘mp_set_int’ is deprecated: replaced by mp_set_u32 [-Wdeprecated-declarations]
    return mpi_to_ltc_error(mp_set_int(a, b));
    ^~~~~~
In file included from src/math/ltm_desc.c:15:0:
../libtommath/tommath.h:318:34: note: declared here
 MP_DEPRECATED(mp_set_u32) mp_err mp_set_int(mp_int *a, unsigned long b);
                                  ^~~~~~~~~~
src/math/ltm_desc.c: In function ‘get_int’:
src/math/ltm_desc.c:98:4: warning: ‘mp_get_int’ is deprecated: replaced by mp_get_mag32/mp_get_u32 [-Wdeprecated-declarations]
    return mp_get_int(a);
    ^~~~~~
In file included from src/math/ltm_desc.c:15:0:
../libtommath/tommath.h:315:54: note: declared here
 MP_DEPRECATED(mp_get_mag32/mp_get_u32) unsigned long mp_get_int(const mp_int *a) MP_WUR;
                                                      ^~~~~~~~~~
src/math/ltm_desc.c: At top level:
src/math/ltm_desc.c:430:13: warning: DIGIT_BIT macro is deprecated, MP_DIGIT_BIT instead
    (int)DIGIT_BIT,
             ^~~~~~
```

It is IMO a bit unfortunate that `mp_set_int` was deprecated, I think such an essential function should be kept for backwards compatibility. Every major bignum library (e.g. GMP https://gmplib.org/manual/Assigning-Integers.html) has something like that for simple C types like `int` or `long`.

I would prefer to keep libtomcrypt in a state when it compiles cleanly with libtommath v1.1.0 as well as with the latest  libtommath/develop.

P.S. I know that this should be perhaps reported/discussed as a libtommath issue but I do not feel comfortable in libtommath's discussions especially when it comes close to `bn_conversion.c` and/or `stdint.h`.

--- a/src/math/ltm_desc.c
+++ b/src/math/ltm_desc.c
@@ -89,13 +89,22 @@ static int init_copy(void **a, void *b)
 static int set_int(void *a, ltc_mp_digit b)
 {
    LTC_ARGCHK(a != NULL);
+#ifdef BN_MP_SET_INT_C
    return mpi_to_ltc_error(mp_set_int(a, b));
+#else
+   mp_set_u32(a, b);
+   return CRYPT_OK;
+#endif
 }
 
 static unsigned long get_int(void *a)
 {
    LTC_ARGCHK(a != NULL);
+#ifdef BN_MP_GET_INT_C
    return mp_get_int(a);
+#else
+   return mp_get_ul(a);
+#endif
 }
 
 static ltc_mp_digit get_digit(void *a, int n)

--- a/src/math/ltm_desc.c
+++ b/src/math/ltm_desc.c
@@ -424,10 +424,14 @@ static int set_rand(void *a, int size)
    return mpi_to_ltc_error(mp_rand(a, size));
 }
 
+#ifndef MP_DIGIT_BIT
+#define MP_DIGIT_BIT DIGIT_BIT
+#endif
+
 const ltc_math_descriptor ltm_desc = {
 
    "LibTomMath",
-   (int)DIGIT_BIT,
+   (int)MP_DIGIT_BIT,
 
    &init,
    &init_copy,

